CONTRIBUTING.md

```diff
@@ -47,7 +47,7 @@ We use Pagure issues to track public bugs. Report a bug by [opening a new issue]
 
 **Notes:** possibly include theories to explain the behaviour, or steps you tried that didn't work or any other information you think that’s valuable that will help our team to provide a fix soon.
 
-People ***love*** thorough bug reports. [This is a good example](https://pagure.io/dogtagpki/issue/3194) of a thorough bug report.
+People ***love*** thorough bug reports. [This is a good example](https://github.com/dogtagpki/pki/issue/3311) of a thorough bug report.
```

Comment:
The link to GH is broken. It should be `https://github.com/dogtagpki/pki/issues/3311`

Reply:
Ah right. It is `/pull/<id>` but `/issues/<id>`...